src/styles/blocks/page-clock.scss

```diff
@@ -0,0 +1,42 @@
+.clockWrapper {
+  display:flex;
+  width:$clock-wrappers-size;
+  height: $clock-wrappers-size;
+  justify-content: center;
+  align-items: center;
+
+  &__cercle {
+    width: $cercle-diametr;
+    height: $cercle-diametr;
+    border-radius: 50%;
+    background-color: aqua;
+    position: relative;
+  }
+
+  &__hour {
+    width: $hands-width;
+    height: $hour-hand-height;
+    background-color: red;
+    @include hands;
+    animation: roll 3600s linear;
+    transform-origin: bottom;
+    animation-iteration-count: infinite;
+  }
+
+  &__minute {
+    width: $hands-width;
+    height: $minute-hand-height;
+    background-color: aqua;
+    @include hands;
+    animation: roll 60s linear;
+    transform-origin: bottom;
+    animation-iteration-count: infinite;
+  }
```

Comment:
You can add animation-iteration-count: infinite; to animation property:
`animation: roll 60s linear infinite;`